src/backend/base/langflow/graph/edge/schema.py

```diff
@@ -32,3 +34,55 @@ def format(self, sep: str = "\n") -> str:
                 for result_pair in self.result_pairs[:-1]
             ]
         )
+
+
+class TargetHandle(BaseModel):
+    fieldName: str = Field(..., description="Field name for the target handle.")
+    id: str = Field(..., description="Unique identifier for the target handle.")
+    inputTypes: Optional[List[str]] = Field(None, description="List of input types for the target handle.")
```

Comment:
`input_types: Optional[List[str]] = Field(None, alias="inputTypes", description="List of input types for the target handle.")`